recipes/astc-codec/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+
+class AstcCodecConan(ConanFile):
+    name = "astc-codec"
+    description = " A software ASTC decoder implementation which supports the ASTC LDR profile"
+    homepage = "https://github.com/google/astc-codec"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    topics = ("conan", "astc", "codec")
+    settings = "os", "compiler", "arch", "build_type"
+    options = {
+        "fPIC": [True, False],
```

Comment:
Why there is no shared option? I see no restriction: https://github.com/google/astc-codec/blob/master/src/decoder/CMakeLists.txt#L17

Reply:
The library doesn't export symbols. And `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` introduces some linking errors.

Reply:
Okay, I didn't find any reference about this error, but I can reproduce it, so I've opened an issue to the author: https://github.com/google/astc-codec/issues/14